lib/src/collect_lcov.dart

```diff
@@ -62,21 +62,20 @@ class LcovCollector {
       this.processSystem: const ProcessSystem(),
       this.sdkRoot}) {}
 
-  Future<String> convertVmReportsToLcov(Directory directoryContainingVmReports,
-      {int workers: 1}) async {
+  Future<String> convertVmReportsToLcov(
+      Directory directoryContainingVmReports) async {
     var reportFiles = await directoryContainingVmReports
         .list(recursive: false, followLinks: false)
         .toList();
 
-    var hitmap = await parseCoverage(reportFiles as Iterable<File>, workers);
+    var hitmap = await parseCoverage(reportFiles as Iterable<File>, null);
```

Comment:
All this plumbing was just for this call, however if you look at the parseCoverage implementation, this parameter is ignored:
https://github.com/dart-lang/coverage/blob/master/lib/src/hitmap.dart#L69